packages/openapi-framework/index.ts

```diff
@@ -260,7 +260,10 @@ export default class OpenAPIFramework implements IOpenAPIFramework {
                 acc[METHOD_ALIASES[method]] = (() => {
                   const innerFunction: any = operation;
                   innerFunction.apiDoc = methodDoc;
-                  return innerFunction;
+                  // Operations get dependencies injected in `this`
+                  return innerFunction.bind({
+                    dependencies: this.dependencies
```

Comment:
also, perhaps we should create a new object, so the operations can't modify the reference?

i.e.

```
  dependencies: {...this.dependencies}
```